### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by dvdschwrtz** [elastos/Elastos.RT#186]
I would like to see more testing in all the repos

**Comment by dvdschwrtz** [comment]
Where is the code for wallet.elastos.org? Is this being rewritten now? Are there other wallets in development?

**Comment by zhaowanli2** [comment]
1.  "I would like to see more testing in all the repos"
We will add the testing codes about Elastos.RT.

**Comment by zhaowanli2** [comment]
The Elastos wallet codes will be added to github on June.

**Comment by jmsaulnier** [comment]
@zhaowanli2 Any news about the the wallet code? Thanks

**Comment by zhaowanli2** [comment]
@jmsaulnier 
1. The sdk for android, this is the github link: https://github.com/elastos/Elastos.RT.SDK.Wallet.Android;
2. The Elasos foundation's wallet app(android), github link:https://github.com/elastos/Elastos.ORG.Wallet.Mobile/tree/wallet_dev;
3. and so on.
Which language would you want to use the elastos wallet?

**Comment by mmate91** [comment]
@zhaowanli2 which is the source code of wallet you are using on wallet.elastos.org? Thanks,Máté

**Comment by zhaowanli2** [comment]
@mmate91 Sorry, it's not an open project.

**Comment by zhaowanli2** [comment]
@dvdschwrtz Sorry, the wallet.elastos.org's codes is not opened.

**Comment by vjsingh-lite** [comment]
@zhaowanli2: Yes, why isn't it open source? Are you planning to make it open source? If yes, by when?

**Comment by mmate91** [comment]
@zhaowanli2  Please answer the question!


@vjsingh-lite

**Comment by bocheng0000** [comment]
wallet.elastos.org is a web wallet for ELA. 
An Android wallet app will be released soon and that wallet is open source.
@mmate91 @vjsingh-lite

**Comment by zhaowanli2** [comment]
wallet.elastos.org is a web wallet for ELA.
An Android wallet app will be released soon and that wallet is open source.
@mmate91 @vjsingh-lite

**Comment by mmate91** [comment]
@zhaowanli2  anyone?

**Comment by mmate91** [comment]
anyone??

**Comment by zhaowanli2** [comment]
Now, it's not an open source project, but it will be open in feature, i think.
